Mysql.sql

```diff
@@ -0,0 +1,7 @@
+CREATE DATABASE Notes_DB;
+CREATE TABLE notes_table ( 
+     AuthorName VARCHAR(100) NOT NULL , 
+     Title VARCHAR(100) NOT NULL ,
+      Note VARCHAR(500) NOT NULL ,
+)
+       
```

Comment:
Why do you not add a primary key?

Reply:
like seppe said you need to put a private key on the title then there can only be one of the same title. if you want more then one title with the same name you should make a column named id and give that a primary key.